doc/modules/ROOT/pages/funcs.adoc

```diff
@@ -1114,6 +1114,44 @@ by passing the result to <<draw-boxes,`draw-boxes`>> or
 (draw-related-boxes (number-as-bits 0xd3 8))
 ----
 
+[[number-as-hex]]
+== number-as-hex
+
+Takes a number and transforms it into a hexadecimal value of the specified
+length.
+
+.Arguments
+[source,clojure]
+----
+[number length]
+----
+
+This can be used as a helper function for generating hexadecimal column 
```

Comment:
I am sorry this stalled, I have a very vivid memory of having added a discussion here that I was waiting to hear back about, but it seems to have disappeared completely. Perhaps it is sitting un-submitted on my other laptop, perhaps GitHub somehow vaporized it! It was not hugely important, so I regret this has been sitting here.

What I said was that I have struggled myself sometimes with the temptation to go down rabbit holes explaining Clojure syntax and details to people, and fear this may be a case where it distracts from the goal of explaining how `number-as-hex` itself works, and perhaps we should stop at “This can be used as a helper function for generating hexadecimal column labels.” If people want to know how `mapv` works, they can look at the Clojure documentation. If we do want to keep this in, the text should be corrected to say the value returned will be a vector, not a list (that’s what the `v` in `mapv` means: the more commonly used `map` function returns a lazy list).

We can also omit the tip explaining the reader macro syntax for anonymous functions by simply not using it. Normal function syntax is not much longer in this case:
```clojure
(mapv (fn [n] (number-as-hex n 2)) (range 16))
```
This is not far from what the reader macro syntax generates, but it has a better variable name.

What do you think?

Reply:
I agree, as I wrote this I felt I was explaining more the nuance of Clojure than describing how to use a library's function. The explicit function call is more readable too.

>If we do want to keep this in, the text should be corrected to say the value returned will be a vector, not a list (that’s what the v in mapv means: the more commonly used map function returns a lazy list).

Of course, not sure how I missed that!

Reply:
Addressed in a9a28cb